File: JavaWebFramework/src/OpenDataAccess/Data/MySqlDataBase.java
package OpenDataAccess.Data;

import java.sql.DriverManager;
import java.sql.SQLException;

public class MySqlDataBase extends DataBase implements IMySqlDataBase {

    MySqlDataBase(String connectionString) {
        this.SetClientType(ServerClient.MySqlClient);
        this.SetConnectionString(connectionString);
    }
}

File: AbetAccount/AbetAccount_Admin/OpenDataAccessCore/OpenDataAccessCore.Data/MySqlDataBase.cs
namespace OpenDataAccessCore.Data
{
    /// <summary>
    /// 访问MySql数据库类
    /// </summary>
    internal sealed class MySqlDataBase : DataBase, IMySqlDataBase
    {
        public MySqlDataBase()
        {
            this.ClientType = ServerClient.MySqlClient;
        }
    }
}
